packages/compass-connections-navigation/src/placeholder-item.tsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable react/prop-types */
```

Comment:
I've seen this disable directive in a couple of files here, it doesn't seem to trigger anything locally though?

Reply:
yup, might not be needed anymore. I'll try removing it and see if it triggers anything in the CI